picodom.d.ts

```diff
@@ -0,0 +1,30 @@
+declare namespace picodom {
+  export interface VNode<D> {
+    tag: string
+    data?: D
+    children: VNode<{}>
+  }
+
+  export interface Component<D> {
+    (data?: D, ...children: VNode<{}>[]): VNode<D>
```

Comment:
This is correct, data is a required argument. It can be `null` or `{}` if you want to ignore it.